### PR TITLE
fix: #24 add filtration by employeeId in HardDeleteEntityCommand so that different employees can't delete each other's records

### PR DESCRIPTION
**Pull request by MDI74**

### CODE REVIEW

File: Application/Commands/HardDeleteEntityCommandTests.cs [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
 [UnitTest]
 public class HardDeleteEntityCommandTests
 {
+    protected const long EMPLOYEE_ID = 1;
+    protected const long TENANT_ID = 777;
+
     private readonly HardDeleteEntityCommand _command;
     private readonly TenantAppDbContext _context;
 
     public HardDeleteEntityCommandTests()
```
Comment (fpandyz) [inline Application/Commands/HardDeleteEntityCommandTests.cs]: Do we need to add a test that I cannot delete an entity of a different employee? I see this check in Karate tests, it looks to me like an edge case check, here we can easily isolate it in the command tests. It looks okeish to keep these tests in Karate but maybe here we should follow this kind of a new rule: if we can move the tests to a deeper layer with less real dependencies and it is sort of the same including its complexity then we should do it.

**Review by fpandyz — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -118,12 +118,14 @@ services:
...
-      AUTH_SECOND_TENANT_PASSWORD_WITH_ALL_PERMISSIONS: "second-tenant-password-with-all-permissions"
-      AUTH_LOGIN_WITHOUT_PERMISSIONS: "first-tenant-login-without-permissions"
-      AUTH_PASSWORD_WITHOUT_PERMISSIONS: "first-tenant-password-without-permissions"
+      AUTH_SLYTHERINE_TENANT_DRACO_MALFOY_LOGIN_WITH_ALL_PERMISSIONS: "AUTH_SLYTHERINE_TENANT_DRACO_MALFOY_LOGIN_WITH_ALL_PERMISSIONS"
+      AUTH_SLYTHERINE_TENANT_DRACO_MALFOY_PASSWORD_WITH_ALL_PERMISSIONS: "AUTH_SLYTHERINE_TENANT_DRACO_MALFOY_PASSWORD_WITH_ALL_PERMISSIONS"
+      AUTH_SLYTHERINE_TENANT_SEVERUS_SNAPE_LOGIN_WITH_ALL_PERMISSIONS: "AUTH_SLYTHERINE_TENANT_SEVERUS_SNAPE_LOGIN_WITH_ALL_PERMISSIONS"
+      AUTH_SLYTHERINE_TENANT_SEVERUS_SNAPE_PASSWORD_WITH_ALL_PERMISSIONS: "AUTH_SLYTHERINE_TENANT_SEVERUS_SNAPE_PASSWORD_WITH_ALL_PERMISSIONS"
+      AUTH_RAVENCLAW_TENANT_CHO_CHANG_LOGIN_WITH_ALL_PERMISSIONS: "AUTH_RAVENCLAW_TENANT_CHO_CHANG_LOGIN_WITH_ALL_PERMISSIONS"
+      AUTH_RAVENCLAW_TENANT_CHO_CHANG_PASSWORD_WITH_ALL_PERMISSIONS: "AUTH_RAVENCLAW_TENANT_CHO_CHANG_PASSWORD_WITH_ALL_PERMISSIONS"
+      AUTH_LOGIN_WITHOUT_PERMISSIONS: "AUTH_LOGIN_WITHOUT_PERMISSIONS"
```
Comment (fpandyz) [inline docker-compose.yml]: Let's rename this guy to follow the convention.
